dts/arm/nxp/nxp_mcxw71.dtsi

```diff
@@ -19,6 +19,7 @@
 
 	chosen {
 		zephyr,bt-hci = &hci;
+		zephyr,nbu = &nbu;
```

Comment:
should be an alias, and not prefixed with `zephyr,` , maybe call the alias `nxp,nbu`